File: C Functions Task/todo2/app4.c
#include <stdio.h>
//called function
void Area(int radius){
    float pi=3.14,result;
    result=pi*radius*radius;
    printf("%f",result);
}

int main()
{
    int r=7;
    //function caller
    Area(r);
    return 0;
}

File: C Functions Task/todo3/app4.java
public class Main
{
	public static void main(String[] args) {
	    int r=7;
	   	System.out.print("Area of circele of radius "+r+" is :");
	   	//function caller
	   	Area(r);
	}
	//called function
	public static void Area(int r){
	   double pi=3.14;
	   double result=pi*(r*r);
	    System.out.println(result);
	}
}
